src/db/h2/lrsql/h2/main.clj

```diff
@@ -10,8 +10,10 @@
   "Run an H2-backed LRSQL instance based on the `:test-h2` (if `:persistent?`
    is set to `true`) or `:test-h2-mem` (if not) config profile. For use with
    `clojure -X:db-h2`."
-  [{:keys [persistent?]}]
-  (let [profile (if persistent? :test-h2 :test-h2-mem)]
+  [{:keys [persistent?
+           override-profile]}]
+  (let [profile (or override-profile
+                  (if persistent? :test-h2 :test-h2-mem))]
     (component/start (system/system h2-backend profile))))
```

Comment:
Maybe you should just have a single `:profile` argument if it's no longer a binary choice.

Reply:
Yeah, the way I did it was specifically to not change the interface as it exists.